src/Elcodi/Component/Sitemap/Command/DumpSitemapCommand.php

```diff
@@ -70,40 +68,23 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this
-            ->sitemapDumper
-            ->dump();
+        $profileName = $input->getArgument('profileName');
+        $dumpers = $this
+            ->sitemapDumperChain
+            ->getSitemapDumpers();
+
+        /** @var SitemapDumper $dumper */
+        $dumper = array_filter($dumpers, function($dumper) use ($profileName) {
```

Comment:
This PR is not valid anymore. We have refactored a bit this class [here](https://github.com/elcodi/elcodi/pull/712)